platform/ui/src/components/Select/Select.mdx

```diff
@@ -0,0 +1,20 @@
+---
+name: Select
+menu: Components
+route: components/select
+---
+
+import { Playground, Props } from 'docz';
+import Select from './';
+
+# Select
+
```

Comment:
Can you add here the `Import` section?

Reply:
Good call. also added in some other components that was missing.